gapic/src/main/com/google/gapid/util/Flags.java

```diff
@@ -40,6 +40,9 @@
   public static final Flag<Boolean> fullHelp =
       value("fullhelp", false, "Print help information incuding hidden flags.", true);
   public static final Flag<Boolean> version = value("version", false, "Print GAPID version.");
+  public static final Flag<Boolean> skipDeviceValidation = value("skip_device_validation", false,
```

Comment:
This does not belong here. Please move it into the `Devices` class.
Also, this should be a hidden flag.

Reply:
Moved to Devices class and changed it to hidden.